packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -585,7 +589,7 @@ export const SpacePlugin = ({
                 invariant(directory, 'No directory selected.');
                 if ((directory as any).queryPermission && (await (directory as any).queryPermission()) !== 'granted') {
                   // TODO(mykola): Is it Chrome-specific?
```

Comment:
Yeah this api is not fully supported across all browsers. Realistically we should disable or hide the action in browsers where the api isn't available

Reply:
https://github.com/dxos/dxos/issues/5337